src/terra-dev-site/home/HomeHero.jsx

```diff
@@ -1,7 +1,7 @@
-import React from 'react';
-import Particles from 'react-tsparticles';
-import imgTerra from './assets/terra-ui.png';
-import styles from './HomeHero.scss';
+import React from "react";
```

Comment:
Linter expects these to all be single quotes. Not sure why Travis didn't complain unless it's turned off.
![Screen Shot 2022-03-04 at 11 48 08 AM](https://user-images.githubusercontent.com/1355485/156815510-98fdaa19-3d0e-4f37-9898-ef22430d6e9a.png)

Reply:
I saw so many linter problems that I ignored them and just let "CTRL+S" do the work. I can fix.